### ISSUE TITLE
Publish image to DockerHub

### ISSUE THREAD

**Issue by ashleybot** [UK-SBCoA/uniform-data-set-dotnet-web#88]
https://github.com/UK-SBCoA/uniform-data-set-dotnet-web/blob/ffe03f57c1a0e526adf780eea40200d23dc71ae0/.github/workflows/container-release.yml#L31

Documentation: https://docs.github.com/en/actions/publishing-packages/publishing-docker-images#publishing-images-to-docker-hub-and-github-packages

**Comment by ashleybot** [comment]
@mlan225 We have a DockerHub org, when you are back, let me know your dockerhub username and I'll add you. https://hub.docker.com/orgs/uksbcoa/members

**Comment by ashleybot** [comment]
Push the image on release

**Comment by mlan225** [comment]
The release branch has been updated and the action to build and push to docker hub has successfully run.

i'll consider getting access to the docker org as part of this issue card, so i'll be standing by for the invite before moving this card to done

**Comment by ashleybot** [comment]
@mlan225 Sent invite to DockerHub UKSBCOA org

**Comment by mlan225** [comment]
Created the workflow, but encountered an issue during the build phase of the workflow.

Some things to note that I have done: 
- Pushed the workflow through main into release in order to run the workflow and test it. Going to switch the workflow activation branch to a PR branch in order to keep from further muddying up the main and release branches with test workflows for this dockerHub push
- Due to my dockerHub account being signed in through my github, I attempted to use a github access token as my password, but that did not seem to work. I switched over to a dockerHub access token as my password and that was able to log me in successfully 
- Modified some of the code form the instruction, changed `on: release:` to `on: push:` since we want to update on a push to release and im not sure we have an actual release workflow as opposed to just pushing to a release branch.
